pynest/nest/raster_plot.py

```diff
@@ -268,7 +268,7 @@ def _make_plot(ts, ts1, node_ids, neurons, hist=True, hist_binwidth=5.0, graysca
 
         plt.bar(t_bins, heights, width=hist_binwidth, color=color_bar, edgecolor=color_edge)
         plt.yticks([int(x) for x in numpy.linspace(0.0, int(max(heights) * 1.1) + 5, 4)])
-        plt.ylabel("Rate (Hz)")
+        plt.ylabel("Rate (spks/s)")
```

Comment:
There seems to be some inconsistency in using () and [] around units. @jessica-mitchell Do we have a policy on this?

Reply:
No we don't; it seems from a quick look into the notation that both parentheses and square brackets are simply style choices - so my opinion is that whatever the majority of cases show that we do that. What do you think @heplesser

Reply:
Spun off to https://github.com/nest/nest-simulator/issues/3375.